cvm/docker-compose.yml

```diff
@@ -123,6 +141,15 @@ configs:
               add_header Content-Type text/plain;
           }
 
+          # Remote attestation service
+          location = /tdx_quote {
+              proxy_pass http://attestation-service:8080/;
```

Comment:
```suggestion
              proxy_pass http://attestation-service:8080;
```

Reply:
Yes right it's in the parent folder indeed. Let's try to have some basic test later to check nginx